### DISCUSSION TITLE
Security headers missed in home page (index) but in subpages it's OK

### DISCUSSION THREAD

**Discussion by rich1n** [vercel/next.js#45563] [Help] [answered]
I have the security headers in the next.config.js file and it's good; I look in the network tabs but all security headers is missed; when I reach /about or /contact there's the security headers doing great!!!

**response headers at HOME page**
Cache-Control: no-store, must-revalidate
Connection: keep-alive
Content-Encoding: gzip
Content-Type: text/html; charset=utf-8
Date: Fri, 03 Feb 2023 20:35:28 GMT
ETag: "7yquv2esn7d1h9"
Keep-Alive: timeout=5
Transfer-Encoding: chunked
Vary: Accept-Encoding
x-nextjs-cache: MISS
X-Powered-By: Next.js

**security headers in other subpage**
Cache-Control: no-store, must-revalidate
Connection: keep-alive
Content-Encoding: gzip
Content-Security-Policy: default-src 'self'; script-src 'self'; child-src lac-eroticvacaction-ui-rich1n-lacapps.vercel.app/; style-src 'self' lac-eroticvacaction-ui-rich1n-lacapps.vercel.app/; font-src 'self';
Content-Type: text/html; charset=utf-8
Date: Fri, 03 Feb 2023 20:35:50 GMT
Keep-Alive: timeout=5
Permissions-Policy: camera=(), microphone=(), geolocation=()
Referrer-Policy: origin-when-cross-origin
Transfer-Encoding: chunked
Vary: Accept-Encoding
X-Content-Type-Options: nosniff
X-Frame-Options: DENY
X-Powered-By: Next.js

**Comment by rich1n** [comment, marked as answer]
```  async headers() {
    return [  
        {
          source: '/(.*)',
          headers: [ security headers ]
```
The problem was in the source path...

```
     source: '/:path*',
        headers: ...
```

**Reply by paulrudy** [reply]
I'm running into this issue as well, but `/:path*` doesn't solve it either. Both that and `/(.*)` appear to work in `vercel dev` locally, but once deployed to Vercel, no such luck.
